src/twisted/python/_setup.py

```diff
@@ -96,6 +96,7 @@
         'pyasn1',
         'cryptography >= 0.9.1',
         'appdirs >= 1.4.0',
+        'pynacl >= 1.0.1',
```

Comment:
This is not mandatory feedback, feel free to push back on this, if you disagree.  I am not really sure about the political environment re: nacl as a crypto lib. Perhaps @reaperhulk or @alex could opine as well, if this is a legitimate concern.